src/main/java/teammates/ui/controller/AdminEmailImageUploadAction.java

```diff
@@ -43,15 +43,14 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         
       
         data.isFileUploaded = true;
-        data.fileSrcUrl = Config.APP_URL + 
-                          Const.ActionURIs.PUBLIC_EMAIL_FILE_SERVE +
+        data.fileSrcUrl = Const.ActionURIs.PUBLIC_EMAIL_FILE_SERVE +
                           "?blob-key=" + 
                           blobKey.getKeyString();
         
-        log.info("New Image Uploaded : " + data.fileSrcUrl);
+        log.info("New Image Uploaded : " + Config.APP_URL + data.fileSrcUrl);
```

Comment:
I just thought of a way to do it. In another issue after this is merged, perhaps?